cinema/views.py

```diff
@@ -118,12 +130,16 @@
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(mixins.ListModelMixin,
+                   mixins.CreateModelMixin,
+                   viewsets.GenericViewSet):
     queryset = Order.objects.prefetch_related(
         "tickets__movie_session__movie", "tickets__movie_session__cinema_hall"
     )
     serializer_class = OrderSerializer
     pagination_class = OrderPagination
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
```

Comment:
The `OrderViewSet` uses `IsAuthenticated` permission class, which is appropriate for allowing authenticated users to create orders. However, ensure that this aligns with the task requirement that only authenticated users can create orders.